src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.NativeRuntime.targets

```diff
@@ -0,0 +1,66 @@
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.NativeRuntime.targets
+
+Contains code to build and link the native runtime at application build time.
```

Comment:
Can we comment here the file is specific to CoreCLR?

Reply:
Let's not do it now, once this is in, I will see how much work would it be to enable static linking for MonoVM (and if this feature is even desired). If it turns out we're not doing it for MonoVM, then I'll update the comment.